R/as_draws_df.R

```diff
@@ -154,9 +146,9 @@ as_draws_df.mcmc.list <- function(x, ...) {
   }
   if (has_iteration_column) {
     iteration_ids <- x[[.iteration]]
-    x[[.iteration]] <- NULL
+    x[.iteration] <- NULL
   } else {
-    iteration_ids <- seq_len(NROW(x))
+    iteration_ids <- 1L:nrow(x)
```

Comment:
What is the reason for this change? `seq_len(n)` is usually preferred over `1L:n` because the former handles `n = 0` consistently.

Reply:
I thought `1L:n` would be slightly faster but `seq_len(n)` is more robust and better choice.